Misc/NEWS.d/next/Tools-Demos/2024-02-16-00-33-14.gh-issue-113299.lqPMAV.rst

```diff
@@ -0,0 +1,3 @@
+The Argument Clinic CLI is now invoked as either :program:`python3
+Tools/clinic` or :program:`python3 Tools/clinic/run_clinic.py`, as
+:file:`Tools/clinic/clinic.py` has been refactored into ``libclinic``.
```

Comment:
The NEWS entry is outdated, there is no ``__main__.py`` script.

Reply:
Ohh, thanks; the NEWS entry was not part of the commit I reverted. Good catch.